feature/experimental/bgp/otg_tests/bgp_tcp_mss_path_mtu/bgp_tcp_mss_path_mtu_test.go

```diff
@@ -0,0 +1,435 @@
+	prot.Enabled = ygot.Bool(true)
+	isis := prot.GetOrCreateIsis()
+	globalISIS := isis.GetOrCreateGlobal()
+	globalISIS.LevelCapability = oc.Isis_LevelType_LEVEL_2
+	globalISIS.Net = []string{fmt.Sprintf("%v.%v.00", dutAreaAddress, dutSysID)}
+	globalISIS.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST).Enabled = ygot.Bool(true)
+	isisLevel2 := isis.GetOrCreateLevel(2)
+	isisLevel2.MetricStyle = oc.Isis_MetricStyle_WIDE_METRIC
+
+	for _, intf := range intfName {
+		isisIntf := isis.GetOrCreateInterface(intf)
+		isisIntf.Enabled = ygot.Bool(true)
+		isisIntf.CircuitType = oc.Isis_CircuitType_POINT_TO_POINT
+		isisIntfLevel := isisIntf.GetOrCreateLevel(2)
+		isisIntfLevel.Enabled = ygot.Bool(true)
+		isisIntfLevelAfi := isisIntfLevel.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST)
+		isisIntfLevelAfi.Metric = ygot.Uint32(200)
+		isisIntfLevelAfi.Enabled = ygot.Bool(true)
+	}
+	gnmi.Replace(t, dut, dutConfIsisPath.Config(), prot)
+}
+
+func configureOTG(t *testing.T, otg *otg.OTG) gosnappi.Config {
+
+	config := otg.NewConfig(t)
+	port1 := config.Ports().Add().SetName("port1")
+
+	iDut1Dev := config.Devices().Add().SetName(atePort1.Name)
+	iDut1Eth := iDut1Dev.Ethernets().Add().SetName(atePort1.Name + ".Eth").SetMac(atePort1.MAC)
+	iDut1Eth.Connection().SetChoice(gosnappi.EthernetConnectionChoice.PORT_NAME).SetPortName(port1.Name())
+	iDut1Ipv4 := iDut1Eth.Ipv4Addresses().Add().SetName(atePort1.Name + ".IPv4")
+	iDut1Ipv4.SetAddress(atePort1.IPv4).SetGateway(dut1Port1.IPv4).SetPrefix(uint32(atePort1.IPv4Len))
+	iDut1Ipv6 := iDut1Eth.Ipv6Addresses().Add().SetName(atePort1.Name + ".IPv6")
+	iDut1Ipv6.SetAddress(atePort1.IPv6).SetGateway(dut1Port1.IPv6).SetPrefix(uint32(atePort1.IPv6Len))
+
+	isisDut1 := iDut1Dev.Isis().SetName("ISIS").SetSystemId(ateSysID)
+
+	isisDut1.Basic().SetIpv4TeRouterId(atePort1.IPv4).SetHostname(isisDut1.Name()).SetLearnedLspFilter(true)
+	isisDut1.Interfaces().Add().SetEthName(iDut1Dev.Ethernets().Items()[0].Name()).
+		SetName("devIsisInt").
+		SetLevelType(gosnappi.IsisInterfaceLevelType.LEVEL_2).
+		SetNetworkType(gosnappi.IsisInterfaceNetworkType.POINT_TO_POINT)
+
+	iDut1Bgp := iDut1Dev.Bgp().SetRouterId(iDut1Ipv4.Address())
+	iDut1Bgp4Peer := iDut1Bgp.Ipv4Interfaces().Add().SetIpv4Name(iDut1Ipv4.Name()).Peers().Add().SetName(atePort1.Name + ".BGP4.peer")
+	iDut1Bgp4Peer.SetPeerAddress(dut1Port1.IPv4).SetAsNumber(ateAS1).SetAsType(gosnappi.BgpV4PeerAsType.EBGP)
+	iDut1Bgp6 := iDut1Dev.Bgp().SetRouterId(iDut1Ipv4.Address())
+	iDut1Bgp6Peer := iDut1Bgp6.Ipv6Interfaces().Add().SetIpv6Name(iDut1Ipv6.Name()).Peers().Add().SetName(atePort1.Name + ".BGP6.peer")
+	iDut1Bgp6Peer.SetPeerAddress(dut1Port1.IPv6).SetAsNumber(ateAS1).SetAsType(gosnappi.BgpV6PeerAsType.EBGP)
+
+	t.Logf("Pushing config to OTG and starting protocols...")
+
+	otg.PushConfig(t, config)
+	otg.StartProtocols(t)
+	return config
+}
+
+func configOTG(t *testing.T, otg *otg.OTG) gosnappi.Config {
+
```

Comment:
no blank line for consistency with other functions

Reply:
updated